lib/spdy/response.js

```diff
@@ -78,8 +78,8 @@ exports.end = function end(data, encoding, callback) {
 exports.push = function push(path, headers, callback) {
   var frame = {
     path: path,
-    method: 'GET',
-    status: 200,
+    method: typeof headers.method === 'string' ? headers.method : 'GET',
```

Comment:
Do we really want to be that restrictive? Perhaps, just `headers.method ? headers.method.toString() : 'GET'`